src/sas/mod.rs

```diff
@@ -0,0 +1,190 @@
+use std::convert::TryFrom;
+
+/// Represents a numeric value that is missing. SAS® supports multiple flavors
+/// of missing values including period (.), underscore (._), and the letters
+/// A through Z (.A - .Z).
+#[repr(u8)]
```

Comment:
I made `MissingValue` a `u8` under the hood. I was surprised how difficult Rust made it to go back and forth, as you'll see down below.